conf/lookup/BCD-Port.csv

```diff
@@ -15520,4 +15518,101 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
+200050,UK,3,KKD,,Kirkcaldy - KKD,Kirkcaldy - KKD,,1,NULL,0,
+200051,UK,3,KLN,,Kings Lynn - KLN,Kings Lynn - KLN,,1,NULL,0,
+200052,UK,3,KWL,Stromness,Kirkwall - KWL,Kirkwall - KWL,,1,NULL,0,
+200053,UK,2,LAS,,Lasham Airport - LAS,Lasham Airport - LAS,,1,NULL,0,
+200054,UK,3,LBK,,Barking - LBK,Barking - LBK,,1,NULL,0,
+200055,UK,3,LEI,,Leith - LEI,Leith - LEI,,1,NULL,0,
+200056,UK,3,LER,Scalloway,Lerwick - LER,Lerwick - LER,,1,NULL,0,
+200057,UK,3,LGP,,London Gateway - LGP,London Gateway - LGP,,1,NULL,0,
+200058,UK,3,LKH,,Lakenheath - LKH,Lakenheath - LKH,,1,NULL,0,
+200059,UK,3,LLA,Caernarfon Port Penrhyn,Llanddulas - LLA,Llanddulas - LLA,,1,NULL,0,
+200060,UK,3,LOS,,Lossiemouth - LOS,Lossiemouth - LOS,,1,NULL,0,
+200061,UK,3,LOW,,Lowestoft - LOW,Lowestoft - LOW,,1,NULL,0,
+200062,UK,2,LYD,,London Ashford (Lydd) Airport - LYD,London Ashford (Lydd) Airport - LYD,,1,NULL,0,
+200063,UK,3,LYH,,Lyneham - LYH,Lyneham - LYH,,1,NULL,0,
+200064,UK,3,MDH,,Mildenhall - MDH,Mildenhall - MDH,,1,NULL,0,
+200065,UK,3,MED,Sheerness Queenborough Ridham Dock Rochester,Medway - MED,Medway - MED,,1,NULL,0,
+200066,UK,3,MIL,Pembroke Olis Epu Ferry,Milford - MIL,Milford - MIL,,1,NULL,0,
+200067,UK,3,MNC,Patington,Manchester - MNC,Manchester - MNC,,1,NULL,0,
+200068,UK,3,MOS,Rhyl Shotton,Mostyn - MOS,Mostyn - MOS,,1,NULL,0,
+200069,UK,3,MWH,,Menwith Hill - MWH,Menwith Hill - MWH,,1,NULL,0,
+200070,UK,3,NGO,,Dollands Moor - NGO,Dollands Moor - NGO,,1,NULL,0,
+200071,UK,3,NIG,,Nigg - NIG,Nigg - NIG,,1,NULL,0,
+200072,UK,3,NOR,Wells,Norwich - NOR,Norwich - NOR,,1,NULL,0,
+200073,UK,3,NPT,Chepstow,"Newport, Gwent - NPT","Newport, Gwent - NPT",,1,NULL,0,
+200074,UK,3,NTH,,Northwich - NTH,Northwich - NTH,,1,NULL,0,
+200075,UK,2,OAK,,Oxford Airport (Kidlington) - OAK,Oxford Airport (Kidlington) - OAK,,1,NULL,0,
+200076,UK,3,PAR,,Par - PAR,Par - PAR,,1,NULL,0,
+200077,UK,3,PEN,Newlyn,Penzance Seaport - PEN,Penzance Seaport - PEN,,1,NULL,0,
+200078,UK,3,PTL,,Portland - PTL,Portland - PTL,,1,NULL,0,
+200079,UK,3,PTM,,Portsmouth - PTM,Portsmouth - PTM,,1,NULL,0,
+200080,UK,3,PUF,,Purfleet - PUF,Purfleet - PUF,,1,NULL,0,
+200081,UK,3,ROS,,Rosyth - ROS,Rosyth - ROS,,1,NULL,0,
+200082,UK,2,RWY,,Ronaldsway Airport (Isle of Man) - RWY,Ronaldsway Airport (Isle of Man) - RWY,,1,NULL,0,
+200083,UK,3,SCA,,Scarborough - SCA,Scarborough - SCA,,1,NULL,0,
+200084,UK,3,SCP,,Scunthorpe - SCP,Scunthorpe - SCP,,1,NULL,0,
+200085,UK,3,STN,Lymington,Southampton - STN,Southampton - STN,,1,NULL,0,
+200086,UK,2,STY,,Stornoway Airport - STY,Stornoway Airport - STY,,1,NULL,0,
+200087,UK,2,SYW,,Sywell AerodromeSYW,Sywell AerodromeSYW,,1,NULL,0,
+200088,UK,3,TEP,,Thames Europort - TEP,Thames Europort - TEP,,1,NULL,0,
+200089,UK,3,THP,,Thamesport Seaport - THP,Thamesport Seaport - THP,,1,NULL,0,
+200090,UK,3,TRT,Althorpe Beckingham Burton Stather Flixborough Wharf Grove Wharves Guness Wharf Keadby Neap House Point Bid Scunthorpe ,Trent - TRT,Trent - TRT,,1,NULL,0,

```

Comment:
Missing Gainsborough 

(Not on the list sent over over but is here:
https://www.gov.uk/government/publications/uk-trade-tariff-freight-location-codes/freight-location-codes#uk-airports)